library/src/jsMain/kotlin/dev/bluefalcon/Bluetooth.kt

```diff
@@ -0,0 +1,13 @@
+package dev.bluefalcon
+
+import kotlin.js.Promise
+
+external class Bluetooth {
+    val referringDevice: BluetoothDevice?
+    fun getDevices(): Promise<List<BluetoothDevice>>
+    fun requestDevice(options: BluetoothOptions?): Promise<BluetoothDevice>
+}
+
+data class BluetoothOptions(
+    val acceptAllDevices: Boolean
+)
```

Comment:
Your library provided inspiration/learnings for a multiplatform project, so, figured I'd return the favor:

The following may prove useful/helpful for you in integrating JavaScript support; it is a bunch of `external` definitions that we've created.

https://github.com/JuulLabs/kable/tree/main/core/src/jsMain/kotlin/external

Reply:
wow very nice!